src/main/java/core/basesyntax/BehaviourFigures.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface BehaviourFigures {
```

Comment:
Consider rename the name of interface, because now it's not related to show behaviour -- `interface Drawable` - that who can be drawn.